lib/server.js

```diff
@@ -39,6 +39,7 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
                 Config.client.variableName,
                 ' = %s;</script>'
               ].join('');
+var TEMPLATE_REDIRECT = '<html><head><meta http-equiv="refresh" content="0;URL=\'%s\'" /></head></html>';
```

Comment:
just checking if we are going with client side redirect or a router redirect. because there could be a genuine redirect within the router.

```js
 <Route path='/' component={App}>
     <Route path='account' component={Account}/>
     <Redirect from='redirect-account' to='something' />
     <Route path='something' component={OldAccount}/>
</Route>
```